sdk/webpubsub/azure-messaging-webpubsub-client/src/main/java/com/azure/messaging/webpubsub/client/ConnectedEvent.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.webpubsub.client;
+
+public class ConnectedEvent {
```

Comment:
Not decided whether we should use `ConnectedEvent`, or just use `ConnectedMessage`.

Also these might be moved to `models` package.